src/Event.h

```diff
@@ -31,6 +36,7 @@ class Event final : public Obj
 	analyzer::ID Analyzer() const { return aid; }
 	EventHandlerPtr Handler() const { return handler; }
 	const zeek::Args& Args() const { return args; }
+	double Time() const { return timestamp; }
```

Comment:
I'm not as deeply in the time management code as you, but I'm noticing that we use "time", "timestamp", and "ts" all to convey timestamps. Might help to try to settle on one.

Reply:
Please address this during merge. As you might guess, my vote is on "timestamp" and "ts" where an abbreviation seems appropriate ;)